pavelib/utils/envs.py

```diff
@@ -240,17 +243,26 @@ def get_django_setting(cls, django_setting, system, settings=None):
         """
         if not settings:
             settings = os.environ.get("EDX_PLATFORM_SETTINGS", "aws")
-        value = sh(
-            django_cmd(
-                system,
-                settings,
-                "print_setting {django_setting} 2>/dev/null".format(
-                    django_setting=django_setting
-                )
-            ),
-            capture=True
-        )
-        return unicode(value).strip()
+        if not os.path.exists(cls.BOK_CHOY_LOG_DIR):
```

Comment:
This is a nit, but this assumes that `PRINT_SETTINGS_LOG_FILE` will include `BOK_CHOY_LOG_DIR`. Probably a safe bet, but perhaps it would be more DRY if we used something like `os.path.dirname(cls.PRINT_SETTINGS_LOG_FILE)` instead, so any changes to the log path could be made in just 1 spot? Feel free to ignore, just a thought.

Reply:
Good point, that was a rushed last-minute addition when the lettuce tests failed without it.  Fixing.